policy/pipeline/required_tasks.rego

```diff
@@ -38,6 +83,9 @@ deny contains result if {
 #   failure_msg: Required task %q is missing
 deny contains result if {
```

Comment:
I think it would be great to not have to duplicate the policy rules. Consider, instead, creating a helper function that returns the list of required tasks from `pipeline-required-tasks` if it's there, otherwise get it from `required-tasks`.